pandas/core/frame.py

```diff
@@ -3835,34 +3835,44 @@ def sortlevel(self, level=0, axis=0, ascending=True, inplace=False,
                                inplace=inplace, sort_remaining=sort_remaining)
 
     def nlargest(self, n, columns, keep='first'):
-        """Get the rows of a DataFrame sorted by the `n` largest
-        values of `columns`.
+        """
+        Return the `n` largest rows sorted by `columns`.
+
+        Sort the DataFrame by `columns` in descending order and return the top
+        `n` rows.
 
         Parameters
         ----------
         n : int
-            Number of items to retrieve
+            Number of items to retrieve.
         columns : list or str
-            Column name or names to order by
+            Column name or names to retrieve values from.
         keep : {'first', 'last'}, default 'first'
             Where there are duplicate values:
-            - ``first`` : take the first occurrence.
-            - ``last`` : take the last occurrence.
+            - `first` : take the first occurrence;
+            - `last` : take the last occurrence.
 
         Returns
         -------
         DataFrame
+            The `n` largest rows in the DataFrame, sorted by the given columns
+            in descending order.
+
+        See Also
+        --------
+        DataFrame.nsmallest : Return the `n` smallest rows sorted by given
+            columns.
 
         Examples
```

Comment:
Can you add an example to differentiate between "first" and "last" `keep` parameters?

Reply:
Nice suggestion. Done.